hpOneView/resources/servers/id_pools_ranges.py

```diff
@@ -0,0 +1,209 @@
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+
+standard_library.install_aliases()
+
+__title__ = 'id-pools-ranges'
+__version__ = '0.0.1'
+__copyright__ = '(C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP'
+__license__ = 'MIT'
+__status__ = 'Development'
+
+from hpOneView.resources.resource import ResourceClient
+
+
+class IdPoolsRanges(object):
+
+    def __init__(self, uri, con):
+        self._connection = con
+        self._client = ResourceClient(con, uri)
+
+    def create(self, resource, timeout=-1):
+        """
+        Creates range.
+
+        A range can be one of two types based upon the range category specified: Generated or Custom. The Generated
+        range type automatically assigns start and end addresses to the range. The Custom range type requires a start
+        address to be specified. The end address may also be specified but is optional.
+
+        Args:
+            resource (dict): Object to create
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Created range
+        """
+        return self._client.create(resource, timeout=timeout)
+
+    def get(self, id_or_uri):
+        """
+        Gets range.
+
+        Using the allocator and collector associated with the range, IDs may be allocated from or collected back to the
+        range.
+
+        Args:
+            id_or_uri: Could be either the range id or uri
+
+        Returns:
+            dict: range
+        """
+        return self._client.get(id_or_uri)
+
+    def enable(self, information, id_or_uri, timeout=-1):
+        """
+        Enables or disables a range.
+
+        Args:
+            information (dict): information to update.
+            id_or_uri: id or uri of range
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Updated resource
+
+        """
+        if id_or_uri:
+            uri = self._client.build_uri(id_or_uri)
+
+        data = {
+            "type": "Range",
+            "enabled": information["enabled"]
+        }
```

Comment:
I think the user should pass the correct dictionary  and the method just pass the `information` to `self._client.update` without modifying it